public/styles/noscript.css

```
@@ -75,6 +75,84 @@
+ * HelsinkiGroteskPro variable font (font file served from CDN).
+ * Regular (400), Medium (500), Bold (700), Black (900), each normal and italic.
+ * Font axes (fvar): wght 200–900, ital 0–1.
+ */
+@font-face {
+  font-display: swap;
+  font-family: HelsinkiGroteskPro;
+  font-style: normal;
+  font-weight: 400;
+  src: url('https://makasiini.hel.fi/delivery/917fc4ffc37c44d31b143244042e0078/9a677fe2c4414b1a40990971cd4ac24c.woff2')
+    format('woff2');
+  text-rendering: optimizeLegibility;
```

Stylelint (17.12.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)